OracleAccessManagement/dockerfiles/12.2.1.4.0/Dockerfile

```diff
@@ -0,0 +1,176 @@
+# # Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
+# #
+# # Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+# #
+# # Author: Kaushik C
+# #
+# ORACLE DOCKERFILES PROJECT
+# --------------------------
+# This is the Dockerfile for Oracle Access Manager 12.2.1.4.0
+#
+# Base image of this dockerfile is the FMW Infrastructure 12.2.1.4.0 docker image.
+#
+# REQUIRED FILES TO BUILD THIS IMAGE
+# ----------------------------------
+# See oam.download file in the install directory
+# fmw_12.2.1.4.0_idm_generic.jar & 
+#
+# Pull base image
+# ---------------
+FROM oracle/fmw-infrastructure:12.2.1.4.0 as base
+#
+# Maintainer
+# ----------
+MAINTAINER OAM Development <Kaushik C>
+#
+#
+# Environment variables required for this build (do NOT change)
+# -------------------------------------------------------------
+ENV FMW_IDM_JAR=fmw_12.2.1.4.0_idm_generic.jar \
+    BASE_DIR=/u01 \
+    ORACLE_HOME=/u01/oracle \
+    PATCH_DIR=/tmp/patches \
+    OPATCH_PATCH_DIR=/tmp/opatch \
+    OPATCH_NO_FUSER=true \
+    SCRIPT_DIR=/u01/oracle/dockertools \
+    PROPS_DIR=/u01/oracle/properties \
+    USER_PROJECTS_DIR=/u01/oracle/user_projects \
+    DOMAIN_ROOT=/u01/oracle/user_projects/domains \
+    DOMAIN_NAME="${DOMAIN_NAME:-oam_domain}" \
+    DOMAIN_HOME="${DOMAIN_ROOT}"/"${DOMAIN_NAME}" \
+    ADMIN_USER="${ADMIN_USER:-}" \
+    ADMIN_PASSWORD="${ADMIN_PASSWORD:-}" \
+    CONNECTION_STRING="${CONNECTION_STRING:-OamDB:1521/orclpdb1.localdomain}" \
+    CONTAINER_DIR=/u01/oracle/user_projects/container \
+    ADMIN_LISTEN_HOST="${ADMIN_LISTEN_HOST:-}" \
+    ADMIN_NAME="${ADMIN_NAME:-AdminServer}" \
+    ADMIN_LISTEN_PORT="${ADMIN_LISTEN_PORT:-7001}" \
+    DOMAIN_TYPE="${DOMAIN_TYPE:-oam}" \
+    RCUPREFIX=${RCUPREFIX:-OAM01} \
+    DB_USER=${DB_USER:-} \
+    DB_PASSWORD=${DB_PASSWORD:-} \
+    DB_SCHEMA_PASSWORD=${DB_SCHEMA_PASSWORD:-} \
+    USER_MEM_ARGS=${USER_MEM_ARGS:-"-Djava.security.egd=file:/dev/./urandom"} \
+    JAVA_OPTIONS="${JAVA_OPTIONS} -Dcom.sun.jndi.ldap.object.disableEndpointIdentification=true" \
+    PATH=$PATH:/usr/java/default/bin:$ORACLE_HOME/oracle_common/common/bin:/u01/oracle/wlserver/common/bin:/u01/oracle/dockertools
+
+#
+# Creation of User, Directories and Installation of OS packages
+# ----------------------------------------------------------------
+USER root
+RUN mkdir -p ${BASE_DIR} && \
+#    useradd -b ${BASE_DIR} -d ${ORACLE_HOME} -m -s /bin/bash oracle && \
+    chmod a+xr ${BASE_DIR} && chown oracle:oracle ${BASE_DIR} && \
+    mkdir -p ${USER_PROJECTS_DIR} && \
+    chown -R oracle:oracle ${USER_PROJECTS_DIR} && chmod -R 775 ${USER_PROJECTS_DIR} && \
+    mkdir -p ${CONTAINER_DIR} && \
+    chown -R oracle:oracle ${CONTAINER_DIR} && chmod -R 775 ${CONTAINER_DIR} && \
+    mkdir -p ${SCRIPT_DIR} && chown oracle:oracle ${SCRIPT_DIR} && \
+    mkdir -p ${PROPS_DIR} && chown oracle:oracle ${PROPS_DIR} && \ 
+    yum install -y libaio && \
```

Comment:
Combine  your `yum` commands into a single command, i.e. `yum -y install libaio hostaname`

Reply:
Will take care of it.